explorer/templates.go

```diff
@@ -303,6 +304,37 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 		"toLowerCase": func(a string) string {
 			return strings.ToLower(a)
 		},
+		"fetchListNavURL": func(groupingStr string, interval int64) string {
+			// This allows all links in years page to be linked to months i.e.
+			// /year -> /months -> weeks -> /days -> /blocks
```

Comment:
I don't follow these comments and what the function should do.  e.g. the switch statement will take a MonthGrouping and set `matchedGrouping = "weeks"`, which is then converted to a time interval value.  Sorry, I'm being dense here...